NEWS

```diff
@@ -1,3 +1,11 @@
+Overview of changes leading to 2.7.1
+Wednesday, Aug 12, 2020
+====================================
+- ot-funcs now handles variable empty glyphs better when hvar/vvar isn't present.
+- Reverted a shaping regression for fonts which lack GDEF.
```

Comment:
The regression happens to fonts with GDEF, the issue that caused the regression was a fix for fonts lacking GDEF table.

Reply:
Oh correct, so does `Reverted a GDEF processing regression.` sound better?